packages/ui/package.json

```diff
@@ -75,7 +75,7 @@
     "@vue/compiler-sfc": "^3.1.4",
     "@vue/eslint-config-standard": "^6.0.0",
     "@vue/eslint-config-typescript": "^7.0.0",
-    "@vue/test-utils": "^2.0.0-beta.14",
+    "@vue/test-utils": "2.0.0-beta.14",
```

Comment:
We need the `ag-grid-community` package as dependency or peer dependency if it is not required in vuestic-ui. 

For now, you have ag-grid-community in our docs package, and because it is monorepo it also available in the UI package.

Reply:
Moved `ag-grid-community, ag-grid-vue3` dependencies to peerDependencies